next.config.js

```diff
@@ -20,8 +21,8 @@ const nextConfig = {
         resourceQuery: { not: /url/ }, // exclude if *.svg?url
         loader: "@svgr/webpack",
         options: {
-          dimensions: false,
-          titleProp: true,
```

Comment:
This should be true, so I'm keeping it there for now. May have to use an alternative to SVGR to get these things in and allow the native <title> element.